doc/manual/nasl/built-in-functions/wmi-functions/wmi_reg_enum_value.md

```diff
@@ -0,0 +1,35 @@
+# wmi_reg_enum_value
+
+## NAME
+
+**wmi_reg_enum_value** - enumerate registry values
+
+## SYNOPSIS
+
+*string* **wmi_reg_enum_value**(wmi_handle: *int*, hive: *int*, key: *string*);
+
+**wmi_reg_enum_value** takes up to 3 positional arguments.
+
+## DESCRIPTION
+
+This function enumerates the registry values.
+
+The named argument *wmi_handle* is an *int* representing a connection to a WMI server. This connection can be opened with on of the **[wmi_connect(3)](wmi_connect.md)** functions.
+
+The optional named argument *hive* is of type *int* and defines which registry hive is used. By default *HKEY_LOCALE_MACHINE* (*2147483650*) is used.
+
+The named argument *key* is a *string* containing the registry key. This is the location of the values to enumerate.
+
+## RETURN VALUE
+
+A multiline *string* containing one value per line or *NULL* on error.
```

Comment:
Not sure about this, it also could be a comma-separated list or something else. I currently have no system to be able to quickly validate this.